src/IntelliHR/Validation/Validators/AbstractValidator.php

```diff
@@ -21,4 +23,20 @@ protected function requireParameterCount($count, $parameters, $rule)
             throw new InvalidArgumentException("Validation rule $rule requires at least $count parameters.");
         }
     }
+
+    /**
+     * @return DateTime|bool
```

Comment:
When does this return a `bool`?

Reply:
PHP returns `false` for some invalid dates, and throws an error for others 😕